code/build/template/include/extensions/orxExtensions.h

```diff
@@ -91,6 +96,10 @@ void InitExtensions()
   // Initialize inspector
   orxInspector_Init();
 ]
+[+python
+  // Initialize Python support
+  orxPy_InitVM();

```

Comment:
Initialize the pocketpy virtual machine. This does not execute any game-specific Python code. It does include all of the `orx.*` engine module bindings.